.travis.yml

```diff
@@ -10,16 +10,29 @@ rvm:
   - 1.9.3
   - 2.0.0
   - 2.1.0
+
 env:
   matrix:
     - PUPPET_GEM_VERSION="~> 3.0.0"
     - PUPPET_GEM_VERSION="~> 3.1.0"
     - PUPPET_GEM_VERSION="~> 3.2.0"
+    - PUPPET_GEM_VERSION="~> 3.3.0"
     - PUPPET_GEM_VERSION="~> 3.4.0"
-    - PUPPET_GEM_VERSION="~> 3.5.1"
+    - PUPPET_GEM_VERSION="~> 3.5.0"
     - PUPPET_GEM_VERSION="~> 3.6.0"
     - PUPPET_GEM_VERSION="~> 3.7.0"
     - PUPPET_GEM_VERSION="~> 3.8.0"
+    - PUPPET_GEM_VERSION="~> 3" PARSER="future"
```

Comment:
I do need to make time to go through all modules and stamp this out.

http://media.tumblr.com/tumblr_m5ozx7iP3x1qcwic6.gif